tests/dragonfly/replication_test.py

```diff
@@ -1208,9 +1208,17 @@ async def seed():
 
     # Give the seeder a bit of time.
     await asyncio.sleep(1)
-    await c_replica.execute_command(f"REPLTAKEOVER 5")
     seeder.stop()
 
+    # Need to wait to synchronize with replica
+    await asyncio.sleep(1)
+
+    c_master = master.client()
+    await c_master.execute_command("SAVE")
+    await disconnect_clients(c_master)
```

Comment:
Not sure if we still need this because it was testing that repltakeover saves a valid file 🤷🏻‍♂️ ... Anyways, doesn't matter

Reply:
Actually we did not test that replicatakeover saves a valid file but that **all** the data that was executed on master was replicated to replica. We did this by starting master again after takeover finished and comparing master and replica data.

Reply:
That is a very important check and I am thinking that it worth doing a save after replicatakeover to be able to test this

Reply:
Ah, sorry, I though we decided to disable this functionality 😅

Reply:
we will add this as optional flag to TAKEOVER command @dranikpg